controllers/comments-controllers.js

```diff
@@ -0,0 +1,15 @@
+const { removeCommentById } = require("../models/comments-models.js");
+const { checkCommentExists } = require("../utils.js");
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+
+  checkCommentExists(comment_id)
+    .then(() => {
+      return removeCommentById(comment_id);
+    })
+    .then(() => {
+      res.status(204).send();
+    })
+    .catch(next);
+};
```

Comment:
I wouldn't say you need to check that the comment exists using a separate query to the database on this endpoint, if it doesn't exist that can be checked as part of your `DELETE` query by using `RETURNING *` at the end of it 💡